stdlib/os/os.go

```diff
@@ -517,3 +518,59 @@ func system(self py.Object, args py.Tuple) (py.Object, error) {
 
 	return py.Int(0), nil
 }
+
+const listDir_doc = `
+Return a list containing the names of the files in the directory.
+
+path can be specified as either str, bytes.  If path is bytes, the filenames
+  returned will also be bytes; in all other circumstances
+  the filenames returned will be str.
+If path is None, uses the path='.'.
+
+The list is in arbitrary order.  It does not include the special
+entries '.' and '..' even if they are present in the directory.
+`
+
+func listDir(self py.Object, args py.Tuple, kwargs py.StringDict) (py.Object, error) {
+	var (
+		path py.Object = py.None
+	)
+	err := py.ParseTupleAndKeywords(args, kwargs, "s:listdir", []string{"path"}, &path)
```

Comment:
if we accept strings or bytes-like objects, I believe the argument should be `s*` instead of `s`.
and as we also accept `None`, I believe it should actually be `z*`.

https://docs.python.org/3.11/c-api/arg.html#strings-and-buffers